### ISSUE TITLE
Issue with Bun upgrade on Windows

### ISSUE THREAD

**Issue by ghost** [oven-sh/bun#10079]
### What version of Bun is running?

1.1.1

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

I went to upgrade my Bun version this morning, in preparation for starting a new project with ElysiaJS. I'm on Bun 1.1.1. and WIndows Powershell 7.4.1.  Here is the error message I received when I did a `bun upgrade`: 

![image](https://github.com/oven-sh/bun/assets/58369164/4ba74a3b-34a0-4c78-892e-3be9361b5c2f)

### What is the expected behavior?

I should have been able to upgrade Bun cleanly to v1.1.3

### What do you see instead?

![image](https://github.com/oven-sh/bun/assets/58369164/4ba74a3b-34a0-4c78-892e-3be9361b5c2f)

### Additional information

Doing a fresh install `powershell -c "irm bun.sh/install.ps1 | iex"` then I'm able to upgrade.

**Comment by ghiscoding** [comment]
I have similar result when trying to update pnpm on Windows, but in my case it's due to permissions on company laptop and that is since they removed local admin access. I'm not saying that is the problem but the output is nearly the same.

**Comment by evaneliasyoung** [comment]
I am experiencing the same issue with **Bun v1.1.3** on **Windows 11 Pro 24H2 26100.1**. Attempting to do the install with sudo or an elevated terminal doesn't make a difference.
```
Bun v1.1.5 is out! You're on v1.1.3

uh-oh: attempt to unwrap error
bun will crash now 😭😭😭

----- bun meta -----
Bun v1.1.3 (2615dc74) Windows x64
UpgradeCommand:
Elapsed: 892ms | User: 46ms | Sys: 46ms
RSS: 80.26MB | Peak: 80.26MB | Commit: 130.80MB | Faults: 19794
----- bun meta -----

0   00007FF7F3ED3E18
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
9   ???

Search GitHub issues https://bun.sh/issues or join in #windows channel in https://bun.sh/discord

thread 42804 panic: attempt to unwrap error
???:?:?: 0x7ff7f2af7ae1 in ??? (bun.exe)
???:?:?: 0x7ff7f2af5589 in ??? (bun.exe)
???:?:?: 0x7ff7f2af8136 in ??? (bun.exe)
???:?:?: 0x7ff7f2b04e38 in ??? (bun.exe)
???:?:?: 0x7ff7f2831603 in ??? (bun.exe)
???:?:?: 0x7ff7f53d3eb3 in ??? (bun.exe)
???:?:?: 0x7ffe0f2b1fd6 in ??? (KERNEL32.DLL)
???:?:?: 0x7ffe10b5d7cf in ??? (ntdll.dll)
```

**Comment by on1force** [comment]
same here, **attempt to unwrap error**
currently on v1.1.4

**Comment by Jarred-Sumner** [comment]
This issue was fixed in Bun v1.1.7 or so, but it will continue to exist for past versions.

You can reinstall Bun by re-running the PowerShell script:
```powershell
powershell -c "irm bun.sh/install.ps1 | iex"
```
